remix-run/react-router-website
# feat (REM-1204): brand page

<img width="600" alt="CleanShot 2022-06-28 at 21 44 48@2x" src="https://user-images.githubusercontent.com/1316441/176346909-e7c9ce6a-7a94-426b-97e9-ecf7ae99d4c7.png">

<img width="300" src="https://user-images.githubusercontent.com/1316441/176346972-fd5c18ce-7122-4689-a7c1-568f08fe4707.jpeg" />



To test:

- Navigate to the page by either:
  - Brand link in footer
  - Right click on "React Router" logo
- Ensure dark/light mode work
- Desktop/Tablet/Mobile

Fixes [REM-1204](https://linear.app/remix-run/issue/REM-1204/make-a-brand-page)
